### PR TITLE
Fix doc links, other minor edits

### PR DESCRIPTION
**Pull request by rpellows**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
 ## Description
 
-The Ocopea orcs component contains all the codebase for building the orchestration components of ocopea and 
-the orcs docker image.
-Visit the ocopea [kubernetes](https://github.com/ocopea/kubernetes) and 
-[cloud foundry](https://github.com/ocopea/cloudfoundry) for installation
+The Ocopea orcs component contains all the codebase for building the orchestration components of Ocopea and 
+the orcs Docker image.
+Visit the Ocopea [Kubernetes](https://github.com/ocopea/kubernetes) and 
+[Cloud Foundry](https://github.com/ocopea/cloudfoundry) for installation
```
Comment (vjtirumalai) [inline README.md]: maybe say "extensions for platform specific installation instructions."

File: README.md [hunk 2]
```diff
@@ -67,17 +68,15 @@ Every pull request must pass the following:
...
-
 ## Licensing
 **{code} does not provide legal guidance on which open source license should be used in projects. We do expect that all
 projects and contributions will have a valid open source license within the repo/project, or align to the appropriate 
 license for the project/contribution.** 
 The default license used for {code} Projects is the [MIT License](http://codedellemc.com/sampledocs/LICENSE "LICENSE").
 
-Ocopea kubernetes extension is freely distributed under the 
+Ocopea Kubernetes extension is freely distributed under the 
 [MIT License](http://emccode.github.io/sampledocs/LICENSE "LICENSE"). See LICENSE for details.
```
Comment (vjtirumalai) [inline README.md]: Need to remove this section as we have our own license.

Comment (vjtirumalai) [reply README.md]: Same is true for Support section, we should remove it.

Comment (rpellows) [reply README.md]: Would like to defer the change to License/Contributing until we have a consistent statement to apply across all projects.  The current PR only addresses minor changes.

Comment (vjtirumalai) [reply README.md]: I added issue to track this. Not sure what you have in mind though. Removal of the sections today makes it consistent with all projects.

**Pull request merged**